src/script/components/list/GroupedConversationHeader.test.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Wire
+ * Copyright (C) 2020 Wire Swiss GmbH
```

Comment:
```suggestion
 * Copyright (C) 2021 Wire Swiss GmbH
```

Reply:
Existing copyrights don't need to be modified.

Reply:
but this is a new file 🤔

Reply:
But it is based on the original code from 2020. git just doesn't see the similarities.